### ISSUE TITLE
Subscribing multiple events at once

### ISSUE THREAD

**Issue by jhorology** [elpheria/rpc-websockets#35]
Now I'm developing a RPC server by Java and it's using more than thousand events.
I would be gratefull if you could consider about following proposal.

```
ws.subscribe("event1", "event2",...,"eventN");
or
ws.subscribe(["event1", "event2",...,"eventN"]);

--> {"jsonrpc": "2.0", "method": "rpc.on", params:["event1", "event2",...,"eventN"], "id": 1}
<-- {"jsonrpc": "2.0", "result": {"event1": "ok", "event2": "ok",...,"eventN": "ok"}, "id": 1}
```

**Comment by mkozjak** [comment]
Hello, @jhorology! This is a nice proposal, yes! It is already handled on my server side. Should be implemented on the client, still. Need to figure out good error handling regarding this.

**Comment by mkozjak** [comment]
The feature is available in 4.3.0.

**Comment by jhorology** [comment]
Really thank you for you so quick work!!
